MAINTAINERS.md

```diff
@@ -6,5 +22,96 @@
 | Peter Nied | [peternied](https://github.com/peternied) | Amazon |
 | Barani Bikshandi | [bbarani](https://github.com/bbarani) | Amazon |
 | Sayali Gaikawad | [gaiksaya](https://github.com/gaiksaya) | Amazon |
+| Abhinav Gupta | [abhinavGupta16](https://github.com/abhinavGupta16) | Amazon |
+| Zelin Hao | [zelinh](https://github.com/zelinh) | Amazon |
+| Yilin Zhang | [zylzulu](https://github.com/zylzulu) | Amazon |
+
+[This document](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md) explains what maintainers do in this repo, and how they should be doing it. If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
+
+## ZenHub Process Workflow
+We follow agile methodologies for our development and release process. We use GitHub issues with annotations via ZenHub to manage and track our stories and issues to effectively manage them over the sprint.
+
+### What is [ZenHub](https://www.zenhub.com/)?
+
+ZenHub is an agile project management and product roadmaps solution, natively integrated into GitHub. It is free to use for opensource repository and comes with a paid membership to manage private repositories. 
+We currently use ZenHub only with our public and opensource repositories.
+
+### Setting up ZenHub
+
+ZenHub can be easily added as an [extension]((https://www.zenhub.com/extension)) to chrome and firefox which can be downloaded for free from the ZenHub website. 
+Alternatively, we can use a the ZenHub [webapp link](https://app.zenhub.com/workspaces/engineering-effectiveness-614cf4272a385f0015d2b48f/board?repos=357723952,406037663) to view the board.
+
+### Sprint Board
+Once the ZenHub extension is installed, ZenHub board can be accessed using the ZenHub tab on GitHub.
+
+![img.png](zenhub_tab_image.png)
```

Comment:
This image seems to be broken on my side.

Reply:
That is strange. The image is added as a part of this PR. Can you verify if you have the image in the root dir?

Reply:
I see then. Thanks!

Reply:
BTW you can look at his fork to see if it would work or not, https://github.com/abhinavGupta16/opensearch-build/blob/abhng-zenhub-doc/MAINTAINERS.md appears fine over there